reactChat/src/App.jsx

```diff
@@ -0,0 +1,139 @@
+import { useState, useEffect } from 'react';
+import AddIcon from '@mui/icons-material/Add';
+import Header from './components/Header.jsx';
+import ChatList from './components/ChatList.jsx';
+import ChatWindow from './components/ChatWindow.jsx';
+import Modal from './components/Modal.jsx';
+
+import './styles/style.css';
+import './styles/chat-list.css';
+import './styles/message-input.css';
+import './styles/chat-styles.css';
+import './styles/modal.css';
+import './styles/search-bar.css';
+import './styles/delete-button.css';
+import './styles/chat-item.css';
```

Comment:
```suggestion
import './styles/index.scss';
```

Было бы лучше использовать один индексный файл для стилей, а в него уже импортировать остальные файлы стилей. Не уверен, что это сработает без SCSS, но внедрить SCSS очень легко:

1. Установить `npm i -D sass`;
2. Переименовать все расширения стилей в `.scss`.